java/server/src/com/appdynamics/wpt/WptHookClient.java

```diff
@@ -28,15 +28,25 @@
 public class WptHookClient {
   private final URL hookReadyUrl;
   private final URL webdriverDoneUrl;
+  private final URL nextActionUrl;
   private final Logger log;
 
   public WptHookClient(String wptHookUrl, Logger log) throws MalformedURLException {
     this.log = log;
     this.hookReadyUrl = new URL(wptHookUrl + "/is_hook_ready");
     this.webdriverDoneUrl = new URL(wptHookUrl + "/event/webdriver_done");
+    this.nextActionUrl = new URL(wptHookUrl + "/event/next_webdriver_action");
+  }
+
+  public void notifyNextWebdriverAction() {
+    try {
+      getResponse(nextActionUrl);
```

Comment:
The hook is not written in a restish way. I'll take care of this later. This doesn't seem to affect the correctness. Is that okay with you?